tests/test-runner/include/Makefile.am

```diff
@@ -1,4 +1,6 @@
 pkgdatadir = $(datadir)/@PACKAGE@/test-runner/include
 dist_pkgdata_SCRIPTS = \
-	logapi.shlib \
+	logapi.shlib
```

Comment:
Question: It appears a lot of our outer `shlib` files don't have the shebang at the top of their files. Does `logapi.shlib` require it for some reason? It's included just like any other `shlib` files.

Reply:
I dunno, I thought that was weird too.

Reply:
My understanding is that it's optional since the `.shlib` files are only ever sourced, but we really should be consistent about it.